### PR TITLE
add takukitamura's profile

### PR DESCRIPTION
**Pull request by TakuKitamura**
# Overview
Add takukitamura's Profile
# Update
- Name
- Year
- Hobbies
- SNS Acount

### CODE REVIEW

File: script/sentence/intro_2.js [hunk 1]
```diff
@@ -4,7 +4,18 @@
...
+## Name \n\
+Taku Kitamura \n\
+## Year \n\
+24 \n\
+## Hobby \n\
+- Taking a photo \n\
+- Programming \n\
+- Running \n\
+## SNS Acount \n\
+- [Twitter](http://qiita.com \"Qiita Home\") \n\
```
Comment (kokada420) [inline script/sentence/intro_2.js:17]: リンクがQiitaになっていますね。

File: script/sentence/intro_2.js [hunk 2]
```diff
@@ -4,7 +4,18 @@
...
 
 const Intro_2_Text = "\
-# YOUR NAME  \n\
+# Profile \n\
+## Name \n\
+Taku Kitamura \n\
+## Year \n\
+24 \n\
+## Hobby \n\
+- Taking a photo \n\
```
Comment (kokada420) [inline script/sentence/intro_2.js:13]: せっかくなので、どのようなカメラを使っているか教えてください。

File: script/sentence/intro_2.js [hunk 3]
```diff
@@ -4,7 +4,18 @@
 /* Plz insert \n\ if you want generate new line.*/
 
 const Intro_2_Text = "\
-# YOUR NAME  \n\
+# Profile \n\
+## Name \n\
+Taku Kitamura \n\
+## Year \n\
+24 \n\
```
Comment (kokada420) [inline script/sentence/intro_2.js:11]: 生年月日も記入してください

File: script/sentence/intro_2.js (hunk 1 above)
Comment (TakuKitamura) [reply script/sentence/intro_2.js]: I fixed it. Please review again.
https://github.com/access-company/ncg2022-github-training-1/pull/19/commits/22595027244e4fa766869d9704af276d8a44e2fc

File: script/sentence/intro_2.js (hunk 2 above)
Comment (TakuKitamura) [reply script/sentence/intro_2.js]: Update about my camera. Please code-review again.
https://github.com/access-company/ncg2022-github-training-1/pull/19/commits/d1e32fc4de6690309b3b55f3b6bb0d155e0844f5

File: script/sentence/intro_2.js (hunk 3 above)
Comment (TakuKitamura) [reply script/sentence/intro_2.js:13]: I add my birthday here.
https://github.com/access-company/ncg2022-github-training-1/pull/19/commits/86041a37b07ff7e3bbaa843e3bd17dd9f94c124f
Please refer this commit.

**Review by kokada420 — APPROVED** [review]
LGTM!

**Pull request merged**